src/utils.py

```diff
@@ -76,8 +78,11 @@ def matrix_visualization(matrix,title=None):
     plt.show()
 
 
-def wavefile_to_waveform(wav_file, features_type):
-    data, sr = sf.read(wav_file)
+def audiofile_to_waveform(audio_file, features_type):
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
```

Comment:
Without silencing warnings, librosa produces a lot of noise when it tries, and fails, to use `soundfile` to open an .mp3, before falling back to another library.